tests/org.jboss.tools.openshift.ui.bot.test/src/org/jboss/tools/openshift/ui/bot/test/connection/v3/OpenNewConnectionWizardTest.java

```diff
@@ -28,50 +27,44 @@
  * Test opening new connection dialog via tool item and context menu.
  * 
  * @author mlabuda@redhat.com
+ * @author adietish@redhat.com
  *
  */
-public class OpenNewConnectionShellTest {
+public class OpenNewConnectionWizardTest {
 	
 	OpenShiftExplorerView explorer = new OpenShiftExplorerView();
 	
 	@Test
-	public void testOpenConnectionShellViaLinkTest() {
+	public void shouldOpenConnectionWizardViaLinkTest() {
+		// given
+		ConnectionUtils.clearAll();
 		explorer.open();
-		
-		try {
-			new DefaultLink(OpenShiftLabel.TextLabels.NEW_CONNECTION).click();
-			// PASS
-		} catch (SWTLayerException ex) {
-			fail("New Connection shell cannot be opened via context menu.");
-		}
-		
-		verifyNewConnectionShellIsOpened();
+
+		// when
+		new DefaultLink(OpenShiftLabel.TextLabels.CREATE_CONNECTION).click();
+
+		// then
+		verifyNewConnectionWizardIsOpened();
 	}
 	
 	@Test
-	public void testOpenConnectionShellViaToolItemTest() {
+	public void shouldOpenConnectionWizardViaToolItemTest() {
+		// given
 		explorer.open();
 		
-		try {
-			new DefaultToolItem(OpenShiftLabel.Others.CONNECT_TOOL_ITEM).click();
-			// PASS
-		} catch (SWTLayerException ex) {
-			fail("New Connection shell cannot be opened via view tool item.");
-		}
-		
-		verifyNewConnectionShellIsOpened();
+		// when
+		new DefaultToolItem(OpenShiftLabel.Others.CONNECT_TOOL_ITEM).click();
+
+		// then
+		verifyNewConnectionWizardIsOpened();
 	}
 	
-	private void verifyNewConnectionShellIsOpened() {
-		Shell connectionShell = null;
-		try {
-			connectionShell = new DefaultShell(OpenShiftLabel.Shell.NEW_CONNECTION);
-		} catch (SWTLayerException ex) {
-			fail("New Connection shell was not opened.");
-		}
-		
+	private void verifyNewConnectionWizardIsOpened() {
```

Comment:
Not sure whether this new way of processing is better. With these changes you wont get a failed test but a test with exception. Previous approach processed exception and provided failure. With this approach you get errors instead of failures, what is not kinda ok.